modules/thermal_hydraulics/test/tests/problems/pressure_drop/gold/pressure_drop_csv_pressure_vpp_FINAL.csv

```diff
@@ -0,0 +1,101 @@
+id,p,x,y,z
+0.005,7000000.2679436,0.005,0,3.0616169978684e-19
```

Comment:
Could you increase the mass flow rate or friction factor to get a large pressure drop?

Reply:
What happened is that in the test I only took 5 very small time steps, so it didn't have time to get close to the end pressure drop. It turned out that I could increase time step size significantly, so this makes the pressure drops much bigger.

Reply:
Ok it looks better know. Making sure we are not capturing noise.